idea/testData/inspectionsLocal/removeRedundantBackticks/.inspection

```diff
@@ -0,0 +1 @@
+org.jetbrains.kotlin.idea.inspections.RemoveRedundantBackticksInspection
```

Comment:
Fixed about this [comment](https://github.com/JetBrains/kotlin/pull/1178#discussion_r131112931)